crates/sargon-uniffi/src/system/sargon_os/delete_account/support/result.rs

```diff
@@ -0,0 +1,8 @@
+use crate::prelude::*;
+use sargon::CreateDeleteAccountManifestResult as InternalCreateDeleteAccountManifestResult;
+
+#[derive(Clone, PartialEq, Eq, InternalConversion, uniffi::Record)]
+pub struct CreateDeleteAccountManifestResult {
```

Comment:
Prefer avoiding "Result" (even in suffix). It is a **very** loaded term, always meaning std::result::Result (or typealiases of that).

I use `Outcome` all over the place in FIP